src/zjs_util.h

```diff
@@ -85,7 +85,7 @@ void zjs_pop_mem_stat(void *ptr);
     ({                                             \
         void *zjs_ptr = zjs_malloc_with_retry(sz); \
         if (!zjs_ptr) {                            \
-            ERR_PRINT("malloc failed\n");          \
+            ERR_PRINT("malloc(%d) failed\n", sz);  \
```

Comment:
space after malloc?

Reply:
That's intended to show the function call, malloc(64) for 64 bytes.